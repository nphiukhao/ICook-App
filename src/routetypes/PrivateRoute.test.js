import React from 'react'
import PrivateRoute from './PrivateRoute'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`PrivateRoute component`, () => {
    it('PrivateRoute snapshot test', () => {
        const tree = shallow(<PrivateRoute/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
})