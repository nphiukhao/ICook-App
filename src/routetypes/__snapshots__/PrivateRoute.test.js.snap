// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute component PrivateRoute snapshot test 1`] = `
<Route
  render={[Function]}
/>
`;
